### DISCUSSION TITLE
Battery Health Grafana Dashboard

### DISCUSSION THREAD

**Discussion by slavag** [teslamate-org/teslamate#3252] [Q&A] [answered]
Hi,
I one of the recent PRs is Battery Health Grafana Dashboard (https://github.com/adriankumpf/teslamate/pull/3059)
Where I can find those dashboards ?
I don't see them in the latest version : 1.27.3 

Thanks

**Comment by cwanja** [comment, marked as answer]
That commit is pushed to the core code, but has not been bundled in a release.

**Comment by CarlosCuezva** [comment]
Here, you can find my version of Battery Health and more

https://github.com/CarlosCuezva/dashboards-Grafana-Teslamate
